internal/data/models.go

```diff
@@ -6,12 +6,15 @@ import (
 )
 
 type Models struct {
+	Payments *PaymentModel
 }
 
 func NewModels(db db.DBConnectionPool) (*Models, error) {
```

Comment:
nit: just realized we might be able to rename DBConnectionPool to ConnectionPool since it's in the db package. Otherwise, we have three `db`s in one line. (nits are non-blocking)

Reply:
Agreed: [fd204fa](https://github.com/stellar/wallet-backend/pull/6/commits/fd204fa7dcbdef92d6369ac6f84d376c2fb341ea)